### PR TITLE
improve(pricefeed): Add default config for CNYUSD to TraderMade

### PR DESCRIPTION
**Pull request by FreshmanQ**
Signed-off-by: FreshmanQ <xunand@quarkchain.org>

**Motivation**

Part of #2479 

**Summary**

Add CNYUSD price identifier default configuration 

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [✅ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

**Comment by FreshmanQ** [comment]
@nicholaspai could you take a look?

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 1]
```diff
@@ -378,6 +378,13 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "bitfinex", pair: "amplusd" }
     ]
   },
+  CNYUSD: {
+    type: "tradermade",
+    pair: "CNYUSD",
+    minuteLookback: 7200,
```
Comment (nicholaspai) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:384]: Curious: what's the lookback limit for the API for the hourly and minutely endpoints? We should add that as a comment somewhere in the TraderMadePriceFeed

**Review by nicholaspai — APPROVED** [review]
LGTM, had one question that might lead to a comment but otherwise looks fine!

**Comment by FreshmanQ** [comment]
> LGTM, had one question that might lead to a comment but otherwise looks fine!

fixed~

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 2]
```diff
@@ -378,6 +378,13 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "bitfinex", pair: "amplusd" }
     ]
   },
+  CNYUSD: {
+    type: "tradermade",
+    pair: "CNYUSD",
+    minuteLookback: 7200, // minuteLookback up to 2 days
+    hourlyLookback: 604800, // hourlyLookback up to 2 months
```
Comment (nicholaspai) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:385]:
```suggestion
    hourlyLookback: 604800, // interval=hour allowed lookback up to 2 months
```

File: packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js [hunk 3]
```diff
@@ -378,6 +378,13 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "bitfinex", pair: "amplusd" }
     ]
   },
+  CNYUSD: {
+    type: "tradermade",
+    pair: "CNYUSD",
+    minuteLookback: 7200, // minuteLookback up to 2 days
```
Comment (nicholaspai) [inline packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js:384]:
```suggestion
    minuteLookback: 7200, // interval=minute allowed lookback up to 2 days
```

**Comment by smb2796** [comment]
Looks like there are merge conflicts that need to be resolved, but once that is done this looks good to me.

**Review by smb2796 — APPROVED** [review]
Looks like there are merge conflicts that need to be resolved, but once that is done this looks good to me.

**Pull request merged**